builtin/logical/pki/cert_util.go

```diff
@@ -985,6 +989,17 @@ func addKeyUsages(data *dataBundle, certTemplate *x509.Certificate) {
 	}
 }
 
+// addPolicyIdentifiers adds certificate policies extension
+//
+func addPolicyIdentifiers(data *dataBundle, certTemplate *x509.Certificate) {
+	for _, oidstr := range data.params.PolicyIdentifiers {
+		oid, err := stringToOid(oidstr)
+		if err == nil {
```

Comment:
I think we should return an error if we can't parse an OID instead of silently swallowing the error.

Reply:
Do you think we should do that at submission time, sanity-checking the role data that was provided? It's safer to do in both places but not sure it's really necessary.